.github/workflows/release-packages.yml

```diff
@@ -0,0 +1,71 @@
+name: Build packages for a release
+
+on:
+  push:
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-linux:
+    name: Build artifacts
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+        - os: macos-latest
+          BUILD_OS_NAME: macos
+
+        - os: ubuntu-latest
+          BUILD_OS_NAME: linux
+
+        - os: ubuntu-latest
+          BUILD_OS_NAME: arm
+
+        - os: windows-latest
+          BUILD_OS_NAME: win64
+
+    steps:
+      - name: Checkout etclabscore/core-geth
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.16
+        if: ${{ matrix.BUILD_OS_NAME != 'arm' }}
+        id: go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.16
+
+      - name: Build all packages (non ARM)
+        if: ${{ matrix.BUILD_OS_NAME != 'arm' }}
+        run: make all
+
+      - name: Build all packages (ARM)
+        uses: pguyot/arm-runner-action@v1
+        id: build_image
+        if: ${{ matrix.BUILD_OS_NAME == 'arm' }}
+        with:
+          image_additional_mb: 2000
+          copy_artifact_path: ./build/bin/
+          copy_artifact_dest: ./build/bin/
+          commands: |
+              wget https://golang.org/dl/go1.16.9.linux-armv6l.tar.gz
+              sudo tar -C /usr/local -xzf go1.16.9.linux-armv6l.tar.gz
+              rm go1.16.9.linux-armv6l.tar.gz
+              export PATH=$PATH:/usr/local/go/bin
+
+              GOARM=5 go run build/ci.go install -dlgo -arch arm
```

Comment:
Should this be `6` to agree with the Go version installed just above?

Reply:
Tricky one for consistency.

`GOARM=5` is copied from https://github.com/etclabscore/core-geth/pull/427/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L78

The installed go above is used to run `go run build/ci.go`, while the `-dlgo` will install the correct needed version. It's a mess indeed. Will check if I can install a better go in the system for `GOARM=5`.

Reply:
Roger. I know very little about ARM architecture versions. Is it worth publishing 5,6,whatever else?

Reply:
I am not familiar with ARM arch too, we can publish them if we like, but maybe it makes sense to do once asked by the community.